src/helm-util.ts

```diff
@@ -51,9 +51,8 @@ export async function getStableHelmVersion(): Promise<string> {
 }
 
 
-export var walkSync = function(dir, filelist, fileToFind) {
-    var files = fs.readdirSync(dir);
-    filelist = filelist || [];
+export const walkSync = function(dir, filelist = [], fileToFind) {
```

Comment:
can this be switched to `export function walkSync(dir, filelist = [], fileToFind) {` to match the other functions in the file?